app/invocation/invocation_execution_util.tsx

```diff
@@ -38,7 +38,7 @@ export function getExecutionStatus(execution: execution_stats.Execution): Execut
     }
     if (execution.exitCode !== 0) {
       return {
-        name: `Failed (exit code ${execution.exitCode})`,
+        name: `Failed`,
```

Comment:
let's remove the other one; seems more consistent with how we're rendering the gRPC status codes above?

Reply:
Sure thing, done